backend/users/views.py

```
@@ -512,141 +577,66 @@ def start_validator_journey(self, request):
             update_user_leaderboard_entries(fresh_user)
 
-            # Transaction successful, return response
             serializer = self.get_serializer(user)
             return Response({
-                'message': 'Builder journey completed successfully!',
+                'message': 'Welcome to GenLayer Builders!',
                 'user': serializer.data
             }, status=status.HTTP_201_CREATED)
-            
+
         except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)